python/python/tests/test_pubsub.py

```diff
@@ -2257,3 +2257,492 @@ async def test_pubsub_context_with_no_callback_raise_error(
 
         with pytest.raises(ConfigurationError):
             await create_two_clients_with_pubsub(request, cluster_mode, pub_sub_exact)
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    async def test_pubsub_channels(self, request, cluster_mode: bool):
+        """
+        Tests the pubsub_channels command functionality.
+
+        This test verifies that the pubsub_channels command correctly returns
+        the active channels matching a specified pattern.
+        """
+        client1, client2 = None, None
+        try:
+            channel1 = "test_channel1"
+            channel2 = "test_channel2"
+            channel3 = "some_channel3"
+            pattern = "test_*"
+
+            client = await create_client(request, cluster_mode)
+            # Assert no channels exists yet
+            assert await client.pubsub_channels() == []
+            await client.close()
```

Comment:
why is it important to close the client?

Reply:
I am pretty sure it shows like we have an open task (similar to what we had with node)

Reply:
but you have the cleanups towards the end of the func?

Reply:
True it's doesn't matter where the close is